src/components/Auth/AuthForm.tsx

```diff
@@ -63,7 +63,7 @@ export const AuthForm: React.FC<Props> = ({ onLogin }) => {
         await loadUser();
       }
     } catch (error) {
-      setErrorMessage('Something went wrtong');
+      setErrorMessage('Something went wrong');
```

Comment:
It's a good practice to use Enum for error messages
```suggestion
      setErrorMessage('Something went wrong');
```